src/pages/Observe/ObserveHeader.tsx

```diff
@@ -0,0 +1,127 @@
+import { AoGateway } from '@ar.io/sdk/web';
+import Button, { ButtonType } from '@src/components/Button';
+import Placeholder from '@src/components/Placeholder';
+import Profile from '@src/components/Profile';
+import {
+  BinocularsGradientIcon,
+  BinocularsIcon,
+  HeaderSeparatorIcon,
+} from '@src/components/icons';
+import { log } from '@src/constants';
+import observationDB from '@src/store/observationsDB';
+import { Assessment } from '@src/types';
+import { performAssessment } from '@src/utils/observations';
+import { useState } from 'react';
+import { Link, useParams } from 'react-router-dom';
+
+const isSearchStringValid = (searchString: string) => {
+  return (
+    searchString.length > 0 &&
+    searchString.split(',').every((s) => s.trim().length > 0)
+  );
+};
+
+const ObserveHeader = ({
+  gateway,
+  setSelectedAssessment,
+}: {
+  gateway?: AoGateway;
+  setSelectedAssessment: React.Dispatch<
+    React.SetStateAction<Assessment | undefined>
+  >;
+}) => {
+  const params = useParams();
+  const ownerId = params?.ownerId;
+
+  const [arnsNamesToSearch, setArnsNamesToSearch] =
+    useState('dapp_ardrive, arns');
```

Comment:
dapp_ardrive is technically an undername, just checking this is ok to do

Reply:
I think so and that it's a good thing to show as a default, but I've sent a message to confirm what defaults to use here.